src/pages/contract/_components/ContractTokensList.tsx

```diff
@@ -0,0 +1,271 @@
+                <FiChevronDown
+                  data-testid="contract-other-tokens-dropdown-icon"
+                  size={24}
+                  className="text-white-700"
+                />
+              ) : (
+                <FiChevronUp
+                  data-testid="contract-other-tokens-up-icon"
+                  size={24}
+                  className="text-white-700"
+                />
+              )}
+            </div>
+            {isTokenIconClicked && (
+              <TokenSearchDropDown
+                data-testid="contract-other-tokens-dropdown"
+                addressTokens={otherTokens.allTokens}
+              />
+            )}
+          </div>
+        </div>
+      </div>
+
+      <div className="flex flex-col md:flex-row mb-9 justify-between md:items-center">
+        <h2
+          data-testid="contract-address-token-list-title"
+          className="font-bold text-xl text-white-50"
+        >
+          {ContractOptionsTitle.Tokens}
+        </h2>
+        <CursorPagination
+          pages={contractTokenListPage}
+          path={`/contract/${id}`}
+          className="justify-end mt-5 md:mt-0"
+        />
+      </div>
+      <div className="hidden lg:block">
+        <div className="grid grid-cols-9 mb-5">
+          <div data-testid="contract-tokens-asset-title">
+            <VerifiedContractSubtitle title={fixedTitle.asset} />
+          </div>
+          <div data-testid="contract-tokens-type-title">
+            <VerifiedContractSubtitle title={fixedTitle.type} />
+          </div>
+          <div data-testid="contract-tokens-symbol-title">
+            <VerifiedContractSubtitle title={fixedTitle.symbol} />
+          </div>
+          <div
+            className="col-span-2 text-right pr-10"
+            data-testid="contract-tokens-amount-title"
+          >
+            <VerifiedContractSubtitle title={fixedTitle.amount} />
+          </div>
+          <div
+            className="text-right pr-5"
+            data-testid="contract-tokens-price-title"
+          >
+            <VerifiedContractSubtitle title={fixedTitle.price} />
+          </div>
+          <div className="text-right" data-testid="contract-tokens-value-title">
+            <VerifiedContractSubtitle title={fixedTitle.value} />
+          </div>
+          <div
+            className="col-span-2 text-right"
+            data-testid="contract-tokens-contract-address-title"
+          >
+            <VerifiedContractSubtitle title={fixedTitle.contractAddress} />
+          </div>
+        </div>
+        <div className="brand-gradient-1 h-[1px]" />
+      </div>
+      {contractTokenList.map((item) => (
+        <TokenRow key={item.contractAddress} data={item} />
+      ))}
+      <CursorPagination
+        pages={contractTokenListPage}
+        path={`/contract/${id}`}
+        className="flex w-full md:justify-end mt-12 md:mt-10"
+      />
+    </div>
+  );
+}
+
+enum ContractOptionsTitle {
+  Code = "Code",
```

Comment:
if we don't need the value of enum, we can leave it empty, e.g.:
```suggestion
  Code,
```